src/components/banner/examples/banner.scss

```diff
@@ -1,10 +1,10 @@
 limel-banner {
-    --icon-background-color: var(--lime-deep-red);
+    --icon-background-color: rgb(var(--color-red-default));
     position: fixed;
     right: 0;
     top: 0;
     left: 0;
-    z-index: 10;
+    z-index: var(--dialog-z-index);
```

Comment:
I don't like the use of `--dialog-z-index` here. It sets a bad example to use a variable that's clearly meant for use in a different component.

I think the banner should have the option to configure the z-index, similar to how that's possible for the dialog. I'll take a look at fixing that, it should be a simple thing to fix, and nice to add a clearly missing (albeit tiny) feature 😊

Reply:
Good point. 100% right!

Reply:
Eeeeh… wtf? _All_ the positioning of the element is actually just done in the example css!?

This is what the "banner" looks like when I remove all the styling from the example, and only the styling that is _actually_ delivered with the component is used:

https://user-images.githubusercontent.com/154725/114392051-8d20bf80-9b98-11eb-9ee6-613dc855f7a6.mov

## 🤦‍♂️

So… I'm not going to do anything major about that… I'll just change the hardcoded value in the example instead. Then we might want to consider reworking the banner component later.

Reply:
Yes, I wanted to tell you about that too. This is super weird and unpractical. But since the focus was on the colors I just updated what we had. Then I forgot about it. We need [a separate issue for it](https://github.com/Lundalogik/crm-feature/issues/1999).